### PR TITLE
ci: Add AMVF to physmon

### PR DESCRIPTION
**Pull request by andiwand**
AMVF is not monitored in the CI at the moment. This PR adds all the checks we have for IVF also to AMVF

### CODE REVIEW

**Comment by andiwand** [comment]
@paulgessinger the AMVF files are kinda big... maybe we should also use the hist root files for the vertexing?

File: CI/physmon/phys_perf_mon.sh [hunk 1]
```diff
@@ -70,20 +70,35 @@ function full_chain() {
...
     Examples/Scripts/generic_plotter.py \
-        $outdir/performance_vertexing_${suffix}.root \
-        vertexing \
-        $outdir/performance_vertexing_${suffix}_hist.root \
+        $outdir/performance_ivf_${suffix}.root \
+        ivf \
+        $outdir/performance_ivf_${suffix}_hist.root \
         --silent \
-        --config CI/physmon/vertexing_config.yml
+        --config CI/physmon/ivf_config.yml
```
Comment (paulgessinger) [inline CI/physmon/phys_perf_mon.sh:77]: Config could be common I guess?

**Comment by paulgessinger** [comment]
Yes ideally we should only keep the histogram files, and I thought that's what we do already for the IVF?

File: CI/physmon/phys_perf_mon.sh (hunk 1 above)
Comment (andiwand) [reply CI/physmon/phys_perf_mon.sh:77]:
```suggestion
        --config CI/physmon/vertexing_config.yml
```

**Comment by andiwand** [comment]
> Yes ideally we should only keep the histogram files, and I thought that's what we do already for the IVF?

you are right. I just messed up

**Review by benjaminhuth — APPROVED** [review]

**Pull request merged**